sdk/textanalytics/Azure.AI.TextAnalytics/src/ErrorCodeValue.cs

```diff
@@ -15,6 +15,8 @@ internal enum ErrorCodeValue
         /// <summary> internalServerError. </summary>
         InternalServerError,
         /// <summary> serviceUnavailable. </summary>
-        ServiceUnavailable
+        ServiceUnavailable,
+        /// <summary> notFound. </summary>
+        NotFound
```

Comment:
this should also be added to https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsErrorCode.cs